core/templates/dev/head/components/forms/schema_editors/SchemaBasedBoolEditorDirective.js

```diff
@@ -27,7 +27,7 @@ oppia.directive('schemaBasedBoolEditor', [
         labelForFocusTarget: '&'
       },
       templateUrl: UrlInterpolationService(
-                   '/forms/schema_editors/Schema_based_bool_editor_directive.html'),
+        '/forms/schema_editors/Schema_based_bool_editor_directive.html'),
```

Comment:
Yes, I didn't realize that there are directives elsewhere. 

I think that we should use only one function (modify getComponentUrl), but we might move the directives from /pages to /components.

Thoughts on this @seanlip?